_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,167 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` [HTML elements][] for which
+
+- the [semantic role][] is the same as the [implicit semantic role][]; and
+- is [included in the accessibility tree][]; and
+- is [visible][].
+
+## Expectation 1
+
+Each test target of type [HTML `<ul>` element][] or [HTML `<ol>` element][], has at least one [HTML `<li>` element][] as a [descendant][] in the [flat tree][] which
```

Comment:
Follow the established writing style please:
- Use proper conditional list formatting
- Don't use angle brackets for elements
- Don't link to HTML elements, they don't need to be disambiguated

Additionally; 
1. Why does a list need to have an `li` element? I get that HTML says so, but I don't see that in WCAG?
2. Why would it need non-empty text? Even HTML doesn't require that?
3. Why can it not be hidden? Why only the hidden attribute, and not some other way to hide it?
4. This leaves unspecified behavior for the dl element. An expectation can't be "inapplicable". It has to be true or false for all test targets. The way this is phrased doesn't allow a true / false answer for dl elements.